app/src/main/java/com/duckduckgo/app/browser/BrowserTrackersAnimatorHelper.kt

```diff
@@ -74,33 +79,92 @@ class BrowserTrackersAnimatorHelper {
         return createLogosViewList(activity, container, logos)
     }
 
-    private fun createLogosViewList(activity: Activity, container: ConstraintLayout, resourcesId: List<Int>?): List<View> {
+    private fun createLogosViewList(
+        activity: Activity,
+        container: ConstraintLayout,
+        resourcesId: List<TrackerLogo>?
```

Comment:
`createResourcesIdList` can return an `emptyList` which could end up here in which case it will return another `emptyList` to `startTrackersAnimation` to skip doing any animation.

Reply:
Exactly, but my point was that an `emptyList` is not nullable. Changing the arg type to `resourcesId: List<TrackerLogo>` will work and be more concrete.

Reply:
Gotcha! I've removed the nullable since it's not needed